1.6/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM centos:centos7
+
+# Jenkins image for OpenShift
+#
+# This image provides a Jenkins server, primarily intended for integration with
+# OpenShift v3.
+#
+# Volumes: 
+# * /var/jenkins_home
+# Environment:
+# * $JENKINS_PASSWORD - Password for the Jenkins 'admin' user.
+
+MAINTAINER Ben Parees <bparees@redhat.com>
+
+ENV JENKINS_VERSION=1.6 \
+    HOME=/var/lib/jenkins \
+    JENKINS_HOME=/var/lib/jenkins
+
+LABEL k8s.io/description="Jenkins is a continuous integration server" \
+      k8s.io/display-name="Jenkins 1.6" \
+      openshift.io/expose-services="8080:http" \
+      openshift.io/tags="jenkins,jenkins16"
+
+# for main web interface:
+EXPOSE 8080
+
+# will be used by attached slave agents:
+EXPOSE 50000
+
+RUN yum install -y wget && \
+  wget -O /etc/yum.repos.d/jenkins.repo http://pkg.jenkins-ci.org/redhat/jenkins.repo && \
+  rpm --import http://pkg.jenkins-ci.org/redhat/jenkins-ci.org.key && \
+  yum install -y tar zip unzip java-1.7.0-openjdk docker jenkins && yum clean all 
+
+#RUN  curl -L -o /tmp/git.hpi http://updates.jenkins-ci.org/latest/git.hpi
+#RUN  curl -L -o /tmp/git-client.hpi http://updates.jenkins-ci.org/latest/git-client.hpi
+#RUN  curl -L -o /tmp/scm-api.hpi http://updates.jenkins-ci.org/latest/scm-api.hpi
+#RUN  curl -L -o /tmp/ssh-credentials.hpi http://updates.jenkins-ci.org/latest/ssh-credentials.hpi
+#RUN  curl -L -o /tmp/credentials.hpi http://updates.jenkins-ci.org/latest/credentials.hpi
+
+RUN  usermod -m -d "$JENKINS_HOME" jenkins && \
+  chown -R jenkins "$JENKINS_HOME" && \
+  chmod o+rw /var/lib/jenkins
+
+RUN wget -O oc.tgz https://github.com/openshift/origin/releases/download/v1.0.1/openshift-origin-v1.0.1-1b60195-linux-amd64.tar.gz && \
+    tar -zxf oc.tgz && \
+    mv oc /usr/local/bin && \
+    rm openshift oadm oc.tgz
+
+COPY run.sh /usr/local/bin/run.sh
```

Comment:
Step 8 : COPY run.sh /usr/local/bin
INFO[0025] stat /var/lib/docker/devicemapper/mnt/52daf88e58d9f363fbb538eef96dd3ee028d4f637ce00a093e4a39e43b5886e2/rootfs/usr/local/bin/run.sh: not a directory